<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit</class>
 <widget class="QDialog" name="Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="back_pushButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>610</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="operation_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>140</width>
     <height>176</height>
    </rect>
   </property>
   <property name="title">
    <string>Operation</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="add_city_radioButton">
      <property name="text">
       <string>Add City</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="delete_city_radioButton">
      <property name="text">
       <string>Delete City</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="add_road_radioButton">
      <property name="text">
       <string>Add Road</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="delete_road_radioButton">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="add_city_groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>191</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Adding city ..</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>X-Axis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_3"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Y-Axis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="add_city_save_pushButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="delete_city_groupBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>171</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>delete city</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="delete_edge_from_comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="delete_city_to_comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="delete_edge_save_pushButton">
      <property name="text">
       <string>save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
